charge.go

```diff
@@ -79,3 +82,12 @@ type Charge struct {
 	// ChargeAuth is the method used to authenticate the charge.
 	ChargeAuth *ChargeAuth `json:"chargeAuth"`
 }
+
+// Sum of kwh for a given hour.
+type kwhPerHour struct {
```

Comment:
Otherwise it will not be accessible outside the package.
```suggestion
type KwhPerHour struct {
```

Reply:
Fixed!